### ISSUE TITLE
Is the label_2 folder under validation an empty folder？

### ISSUE THREAD

**Issue by wutaoo** [qianguih/voxelnet#22]
if not, where can i get

**Comment by zemahran** [comment]
No, it contains 7480 training labels provided with the KITTI dataset. You can get them from [here](http://www.cvlibs.net/datasets/kitti/eval_object.php?obj_benchmark=3d).

**Comment by tiandaji** [comment]
I've read the readme several times, and I have the same doubt as @wutaoo 
The instruction says, we should download these following zip files:

1. Velodyne point clouds (29 GB): input data to VoxelNet
2. Training labels of object data set (5 MB): input label to VoxelNet
3. Camera calibration matrices of object data set (16 MB): for visualization of predictions
4. Left color images of object data set (12 GB): for visualization of predictions

For the 2nd one, however, there is only one training folder, not the same as the others with both training and testing folders, which leads to the data lacking problem:

└── DATA_DIR
       ├── training   <-- training data
       |   ├── image_2
       |   ├── label_2
       |   └── velodyne
       └── validation  <--- evaluation data
       |   ├── image_2
       |   ├── label_2 **(have a little problem, do not have these data)**
       |   └── velodyne

So I am wondering how we update the testing data file path for label_2 in the launch_test.sh to replace "#GT_DIR=/media/hdc/KITTI/for_voxelnet/cropped_dataset/validation/label_2".

From your previous post on May 27 @zemahran , I assume that you meant to copy the training label_2 as the validating data.  Maybe I misunderstood or missed something important in the instructions, thanks a lot if you guys can give me some indications @zemahran @qianguih

**Comment by StephenDBW** [comment]
I have solved this problem. The cause of the problem is that the label_2 folder in the validation set is empty.You must manually divide the KITTI data into training and validation.
